Code/IO/include/sitkImageFileReader.h

```diff
@@ -55,6 +55,12 @@ namespace itk {
      * when multiple ImageIOs "can read" the file and the user wants
      * to select a specific IO (not the first).
      *
+     * \note DICOM tags represent hexadecimal numbers,
```

Comment:
Please begin with stating how things ARE. e.g. DICOM tags are index as strings in the meta-data dictionary(s), therefore "0020|000D" and "0020|000d" are different when accessing the tag.

When setting a tag does the capitalization matter?

Reply:
When setting the tag the capitalization doesn't matter. Writing is done correctly no matter if you set the tag via "0020|000D" or "0020|000d". 

I would recommend to always use lower case, because of the inconsistency that occurs when setting the tag using upper case and then accessing it (it is upper case in the meta-data dictionary) vs. setting the tag using upper case, writing to disk, reading from disk and then accessing it (it will be lower case in the meta-data dictionary).

Reply:
Language updated.